tests/cypress/e2e/test_demos.cy.js

```diff
@@ -0,0 +1,37 @@
+describe( 'Test CKEditor 5 demo', () => {
+  const defaultDemos = [ 
+    'feature-rich',
+    'headless',
+    'internationalization',
+    'markdown',
+    'mobile',
+    'source-code-editing',
+    'user-interface-balloon',
+    'user-interface-balloon-block',
+    'user-interface-bottom-toolbar',
+    'user-interface-button-grouping',
+    'user-interface-classic',
+    'user-interface-document',
+    'user-interface-inline'
+  ]
+
+  defaultDemos.forEach( demo => {
+    it( `Testing demo: ${ demo }`, () => {
+      const URL = `http://localhost:9001/${ demo }/`;
+
+      cy.visit( URL );
+
+      if ( demo == 'mobile' ) {
+        cy.get( 'iframe' ).then( $iframe => {
+          const doc = $iframe.contents();
+
+          doc.get( '.ck-editor__editable' );
+        } )
+        
+        return;
+      }
+
+      cy.get( '.ck-editor__editable' );
```

Comment:
Here we could also check if the images are correctly loaded. If I recall correctly, the content of demos on ckeditor.com was moved to the private repo, thus websites e2e tests would not catch an issue with images here.  The following snippet should do the job:
```
	if ( Cypress.$( 'img' ).length > 0 ) {
			cy.get( 'img' ).each( $img => {
				expect( $img[ 0 ].naturalWidth ).to.be.above( 0 );
			} );
		}
```
it will fail if the image was not correctly fetched and shows as a broken image instead.

Reply:
Good idea, I'll add this check, thanks!